python/pipeline/.ipynb_checkpoints/exceptions-checkpoint.py

```diff
@@ -0,0 +1,6 @@
+class PipelineException(Exception):
```

Comment:
Can you delete this .ipynb_checkpoints from the PR?

Reply:
Deleted. Somehow some of the `.ipynb_checkpoints` are not completed ignored even after putting into `.gitignore`